typescript/api/controllers/webservice/RecordController.ts

```diff
@@ -256,6 +257,36 @@ export module Controllers {
         });
     }
 
+    public getRecordAudit(req, res) {
+      const brand = BrandingService.getBrand(req.session.branding);
+      var oid = req.param('oid');
+      var dateFrom = req.param('dateFrom');
+      var dateTo = req.param('dateTo');
+      let params = {'oid': oid, 'dateFrom': null, 'dateTo': null};
+      if(!_.isEmpty(dateFrom)) {
+        params['dateFrom'] =  new Date(dateFrom);
+      }
+
+      if(!_.isEmpty(dateTo)) {
+        params['dateTo'] =  new Date(dateTo);
+      }
+
+      Observable.fromPromise(this.RecordsService.getRecordAudit(params)).subscribe(audit => {
+
+        let found = 0;
+        if(_.isArray(audit)) {
+          found =  audit.length;
+        }
+
+        let ret= {"Type": "Record Audit", "Search Parameters": params, "Found": found, "data": audit}
```

Comment:
We have a standard API response type for lists https://github.com/redbox-mint/redbox-portal/blob/develop/core/src/model/ListAPIResponse.ts